ntoskrnl/ex/sysinfo.c

```diff
@@ -664,7 +664,11 @@ QSI_DEF(SystemProcessorInformation)
 #else
     Spi->MaximumProcessors = 0;
 #endif
+#if (NTDDI_VERSION >= NTDDI_WIN10) || ((NTDDI_VERSION >= NTDDI_WIN8) && defined(_WIN64))
```

Comment:
See my previous comments: https://www.geoffchappell.com/studies/windows/km/ntoskrnl/inc/ntos/amd64_x/kprcb/index.htm appears to indicate Win8.1 i.e. `NTDDI_WINBLUE` instead of Win8.